### ISSUE TITLE
Jazzy debian package release

### ISSUE THREAD

**Issue by gregtkogut** [PlanSys2/ros2_planning_system#333]
I don't know if this falls under the scope of the PlanSys2 maintainers, but is there a known time that ROS 2 jazzy packages will be released to `main`: https://repo.ros2.org/status_page/ros_jazzy_default.html?q=plansys2 ?

The immediate symptom is that `apt install ros2-jazzy-plansys2-core` isn't working on the `osrf/ros:jazzy-desktop-full` docker image.

**Comment by fmrico** [comment]
Ok, thanks. I have to check It. And yes, It is my responsability 😕

**Comment by emanuelazfernandes** [comment]
@fmrico it seems like the problem in the failed build (action workflow) is related to the adding of rqt_gui_cpp/include/plugin.hpp when it used to be rqt_gui_cpp/include/plugin.h
From the commit history in the ros_visualization/rqt repo, it seem like this was added only for rolling and not for jazzy.
If it would be useful, I could create a PR but I don't think I have permission to trigger the action to verify and test.

**Comment by fmrico** [comment]
> [@fmrico](https://github.com/fmrico) it seems like the problem in the failed build (action workflow) is related to the adding of rqt_gui_cpp/include/plugin.hpp when it used to be rqt_gui_cpp/include/plugin.h From the commit history in the ros_visualization/rqt repo, it seem like this was added only for rolling and not for jazzy. If it would be useful, I could create a PR but I don't think I have permission to trigger the action to verify and test.

No, the problem was related to CI. I was checking out `rolling` to test the branch `jazzy-devel`. See https://github.com/PlanSys2/ros2_planning_system/commit/3fc9e946067c75169772851c5d762d323efd5383
